activiti-spring-boot-starter/src/test/resources/processes/JobEventsTest.testJobEntityEvents.bpmn20.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Are you using this process?

Reply:
This process is not used currently, just put it because it has some behavior not covered in tests...

Reply:
anticipation should be avoided

Reply:
Removed